cloud/pkg/controller/controller.go

```diff
@@ -9,12 +9,10 @@ import (
 	"github.com/kubeedge/kubeedge/cloud/pkg/controller/config"
 	"github.com/kubeedge/kubeedge/cloud/pkg/controller/constants"
 	"github.com/kubeedge/kubeedge/cloud/pkg/controller/controller"
-	"github.com/kubeedge/kubeedge/cloud/pkg/controller/messagelayer"
 )
 
 // Controller use beehive context message layer
 type Controller struct {
```

Comment:
Controller will be empty structure now. Can you open and close the curly brackets at the same line please ?

Reply:
ok